src/lib/model/InboundTransfersModel.js

```diff
@@ -244,6 +275,45 @@ class InboundTransfersModel {
         }
     }
 
+    /**
+     * Validates  an incoming transfer prepare request and makes a callback to the originator with
+     * the result
+     */
+    async executeTransactionRequest(transactionRequest, sourceFspId) {
```

Comment:
@ksatya77 What is the puspose of executeTransactionRequest function? This looks like duplicate of transactionRequest functino in this file?

Reply:
ohh yes...copy paste issue...removed the duplicate now